cura/MachineManagerModel.py

```diff
@@ -486,6 +486,13 @@ def hasMaterials(self):
         return False
 
     @pyqtProperty(bool, notify = globalContainerChanged)
+    def hasHeatedBed(self):
+        if self._active_container_stack:
+            return bool(self._global_container_stack.getMetaDataEntry("machine_heated_bed", False))
```

Comment:
Also note that the global container stack may be None if there is no printer yet (i.e. the user is still in initial set-up).